ui/src/reusable_ui/components/wizard/WizardController.tsx

```diff
@@ -19,50 +19,66 @@ interface Props {
   children: Array<ReactElement<WizardStepProps>>
   handleSkip?: () => void
   skipLinkText?: string
+  jumpStep?: number
 }
 
 @ErrorHandling
 class WizardController extends PureComponent<Props, State> {
   public static defaultProps: Partial<Props> = {
-    skipLinkText: 'Skip',
+    skipLinkText: 'skip',
+    jumpStep: -1,
   }
 
   public static getDerivedStateFromProps(props: Props, state: State) {
     let {currentStepIndex} = state
     const {children} = props
-
     const childSteps = React.Children.map(
       children,
       (child: ReactElement<WizardStepProps>, i) => {
         const isComplete = child.props.isComplete()
+        let isErrored
+        if (typeof child.props.isErrored === 'boolean') {
```

Comment:
what do you think of using the lodash `_.isBoolean`?

Reply:
that's cool I didn't think typescript would get that.